e2etests/test/device_acknowledge_tests.js

```diff
@@ -61,19 +90,24 @@ function device_acknowledgment_tests (deviceTransport, createDeviceMethod) {
     });
 
     it('Service sends 1 C2D message and it is re-sent until completed', function (done) {
+      this.timeout(15000);
```

Comment:
>15000 [](start = 19, length = 5)

that's very short we usually go for 60 seconds

Reply:
I've done thousands of runs on these tests.  This timeout was triggered except for when there was "loss" of a message and it was never going to come in.  Note that this this is on the individual tests.